rotkehlchen/globaldb/handler.py

```diff
@@ -135,6 +136,22 @@ def __new__(
         GlobalDBHandler.__instance.conn = _initialize_global_db_directory(data_dir, sql_vm_instructions_cb)  # noqa: E501
         return GlobalDBHandler.__instance
 
+    @staticmethod
+    def packaged_db_conn() -> DBConnection:
+        """Return a DBConnection instance for the packaged global db."""
+        if GlobalDBHandler() is not None:
```

Comment:
Why did you add this check here? Is not needed. Also for the record if you initialize the globaldb for the first time without the needed arguments it will raise an error